addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -80,6 +80,10 @@ insertModelFields('mail.tracking.value', {
 });
 insertModelFields('res.partner', {
     description: { string: 'description', type: 'text' },
+    user_ids: { string: "Users", type: "one2many", relation: 'res.users', default:[] },
```

Comment:
Are you sure this field is necessary here in master?

Note: fields are automatically fetched from the server now. Defining them here is only necessary for "fake" fields or to override the default value.